VHDLFormatter.js

```diff
@@ -342,7 +341,7 @@ function beautify(input, settings) {
     input = input.replace(/([a-zA-Z0-9\); ])\);(@@comments[0-9]+)?@@end/g, '$1\r\n);$2@@end');
     input = input.replace(/[ ]?([&=:\-<>\+|\*])[ ]?/g, ' $1 ');
     // Do not add space around unary minus or plus
-    input = input.replace(/(, |= |\(|\bRANGE\b |\bRETURN\b |\bIF\b |\bWHILE\b |\bCASE\b |\bIN\b |\bTO\b |\bDOWNTO\b )[ ]*([\-+])[ ]*(\S)/g, '$1$2$3');
+    input = input.replace(/(, |= |\(|\bRANGE\b |\bRETURN\b |\bIF\b |\bELSE\b |\bWHILE\b |\bCASE\b |\bWHEN\b |\bIN\b |\bTO\b |\bDOWNTO\b )[ ]*([\-+])[ ]*(\S)/g, '$1$2$3');

```

Comment:
What does this part `input = input.replace(/(, |= |\(|\bRANGE\b |\bRETURN\b |\bIF\b |\bELSE\b |\bWHILE\b |\bCASE\b |\bWHEN\b |\bIN\b |\bTO\b |\bDOWNTO\b )[ ]*([\-+])[ ]*(\S)/g, '$1$2$3');` fix?

Can you provide an example of this usage?

Reply:
When using the fixed point package it's quite common to have negative numbers around. Normally the code formatter adds a space to both sides of a '-' or '+' symbol, but if there is a single operand, it might be preferable to add space only to its left.

Let's take the following lines as an example:
``` vhdl
signal x, y: sfixed(4 downto -18);
x <= to_sfixed(-2.3, x);
with x select y <= 
    x when -2,
    resize(x + 1, 4, -18) when others;
```
They are turned into something like the following by the formatter:
``` vhdl
signal x, y: sfixed(4 downto - 18);
x <= to_sfixed( - 2.3, x);
with x select y <= 
    x when - 2,
    resize(x + 1, 4, - 18) when others;
```
Basically, what I'm trying to do with that regex is remove the space after `+` or `-` if they come after `,`, `(` or a keyword. The list is not exhaustive for the keywords, and just now I notice that `|` could also be a symbol to look for. Also, it would probably be nicer to use the keyword list you already have in there instead of making another one, but I was trying to get the quickest solution running so I didn't do that.

If you think this is a taste issue, it would still be nice to have it as an option.

Reply:
it is absolutely an improvement. I'll move this part into ts files. I've put your name explicitly in the README.md file of this repo. Thanks for your suggestion.

Reply:
Yes, but there are also some separators like `,`, `(` and `|` which need to be taken into account.